tests/test_generate.py

```diff
@@ -258,20 +262,35 @@ def test_generate(yaml_dir, tmpdir):
     t3 = html_blob.updated
     assert t2 != t3
 
-    # Force generation of Python docs and verify timestamp
+    # Force generation of Python docs and verify timestamp.
     language = "python"
     generate.build_language_docs(test_bucket, language, credentials)
     html_blob = bucket.get_blob(html_blob.name)
     t4 = html_blob.updated
     assert t3 != t4
 
-    # Force generation of Go docs, verify timestamp does not change
+    # Force generation of Go docs, verify timestamp does not change.
     language = "go"
     generate.build_language_docs(test_bucket, language, credentials)
     html_blob = bucket.get_blob(html_blob.name)
     t5 = html_blob.updated
     assert t4 == t5
 
+    # Force regeneration of a single doc with old YAML and verify
+    # timestamp does not change.
+    generate.build_new_docs(test_bucket, credentials)
```

Comment:
This comment and `build_new_docs` call don't line up.

Reply:
done!